README.md

```diff
@@ -1,14 +1,46 @@
 # todo-trello
-Todo app, based on Trello, using `pytrello2`
+Todo-Trello is a simple task management application based on Trello, using `pytrello2`. It allows users to organize their tasks on boards, create lists, and manage cards with ease.
 
 ![](./assets/todo-trello.png)
 
+## Features
```

Comment:
apart from Boards and Cards, our application doesn't support the other features, should we remove them?